pygmt/src/fitcircle.py

```diff
@@ -0,0 +1,127 @@
+"""
+fitcircle - Fit coordinators to create vectors on a sphere.
+"""
+import warnings
+
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile, build_arg_string, fmt_docstring, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    L="normalize",
+    S="small_circle",
+    V="verbose",
+)
+def fitcircle(data, output_type="pandas", outfile=None, **kwargs):
+    r"""
+    Fit coordinators to create vectors on a sphere.
+
+    **fitcircle** reads lon,lat [or lat,lon] values from the first two
+    columns on standard input [or *table*]. These are converted to
+    Cartesian three-vectors on the unit sphere. Then two locations are
+    found: the mean of the input positions, and the pole to the great circle
+    which best fits the input positions. The user may choose one or both of
+    two possible solutions to this problem. When the data are closely grouped
+    along a great circle both solutions are similar. If the data have large
+    dispersion, the pole to the great circle will be less well determined
+    than the mean. Compare both solutions as a qualitative check.
+
+    Setting `normalize` to **1** approximates the
+    minimization of the sum of absolute values of cosines of angular
+    distances. This solution finds the mean position as the Fisher average
+    of the data, and the pole position as the Fisher average of the
+    cross-products between the mean and the data. Averaging cross-products
+    gives weight to points in proportion to their distance from the mean,
+    analogous to the "leverage" of distant points in linear regression in
+    the plane.
+
+    Setting `normalize` to **2** approximates the
+    minimization of the sum of squares of cosines of angular distances. It
+    creates a 3 by 3 matrix of sums of squares of components of the data
+    vectors. The eigenvectors of this matrix give the mean and pole
+    locations. This method may be more subject to roundoff errors when there
+    are thousands of data. The pole is given by the eigenvector
+    corresponding to the smallest eigenvalue; it is the least-well
+    represented factor in the data and is not easily estimated by either
+    method.
+
+    Full option list at :gmt-docs:`fitcircle.html`
+
+    {aliases}
+
+    Parameters
+    ----------
+    output_type : str
+        Determine the format the xyz data will be returned in [Default is
+        ``pandas``]:
+
+            - ``numpy`` - :class:`numpy.ndarray`
+            - ``pandas``- :class:`pandas.DataFrame`
+            - ``file`` - ASCII file (requires ``outfile``)
+    outfile : str
+        The file name for the output ASCII file.
+    normalize : float or bool
```

Comment:
Because this parameter specifies the *norm*. I think it makes more sense to alias it to `norm`, rather than `normalize`.
```suggestion
    norm : int or bool
```